config/test_cases/test_pull_request_openshift_ansible_tox.yml

```diff
@@ -18,6 +18,20 @@ actions:
     repository: "openshift-ansible"
     script: |-
       tox
+  - type: "script"
+    title: "build images for integration tests"
+    repository: "openshift-ansible"
+    script: |-
+      if [ -x test/integration/build-images.sh ]; then
```

Comment:
It also checks for existence.
http://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html
So it silently passes if the file is missing.